.eslintcache

```diff
@@ -1 +1 @@
\ No newline at end of file
+[{"/Users/bdizzle/Desktop/codebases/4-mod/fee-dreamhome/src/components/Sourvey/Sourvey.tsx":"1"},{"size":934,"mtime":1609981698141,"results":"2","hashOfConfig":"3"},{"filePath":"4","messages":"5","errorCount":1,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"13szocz","/Users/bdizzle/Desktop/codebases/4-mod/fee-dreamhome/src/components/Sourvey/Sourvey.tsx",["6"],{"ruleId":null,"fatal":true,"severity":2,"message":"7","line":1,"column":0},"Parsing error: Merge conflict marker encountered."]
```

Comment:
Does this change every time you have a new person driving?

Reply:
we are not doing driver/navigator but this changes every time we checkout to a new or existing branch 😭